bin/riemann-tail

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+# Reports running process count to riemann.
+
+require File.expand_path('../../lib/riemann/tools', __FILE__)
+require 'date'
+
+class Riemann::Tools::Proc
+  include Riemann::Tools
+
+  opt :service_name, "name of the service you'll feed to this process.", type: :string
+  opt :csv_mode, "in this mode, we consume a streaming csv of riemann event fields.", default: false
```

Comment:
Hmm, come to think of it--if this is gonna be for arbitrary logfiles, we're gonna need a lot of different format specifiers. Maybe the best thing is to have a `--format csv` option, and add more as we go along?